### PR TITLE
fix(SSheetDialog): onclose stopPropagation

### PR DESCRIPTION
**Pull request by zyz**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor [hunk 1]
```diff
@@ -78,10 +78,12 @@
     {
         if (ValueChanged.HasDelegate)
         {
+            Console.WriteLine("SSheetDialog HasDelegate OnClose");
```
Comment (doddgu) [inline src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor:81]: Remove the debug info.

File: src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor [hunk 2]
```diff
@@ -78,10 +78,12 @@
...
             await ValueChanged.InvokeAsync(false);
         }
         else
         {
+            Console.WriteLine("SSheetDialog OnClose");
```
Comment (doddgu) [inline src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor:86]: Remove the debug info.

**Review by doddgu — CHANGES_REQUESTED** [review]

**Comment by doddgu** [comment]
@capdiem PTAL

File: src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor [hunk 3]
```diff
@@ -2,17 +2,17 @@
 
 <MBottomSheet ContentClass="m-sheet-dialog full-height elevation-0 m-dialog--fullscreen" Value=Value ValueChanged="ValueChanged">
     <MHover>
-        <div @attributes="context.Attrs" style="@($"padding-right:{Padding}px;padding-left:{Padding}px")" class="m-sheet-dialog-overlay d-flex flex-column full-height" @onclick="OnClose">
+        <div @attributes="context.Attrs" style="@($"padding-right:{Padding}px;padding-left:{Padding}px")" class="m-sheet-dialog-overlay d-flex flex-column full-height" @onclick="OnClose" stopPropagation>
```
Comment (capdiem) [inline src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor:5]: `@onclick:stopPropgation`

File: src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
             @if (HeaderContent is null)
             {
-                <div @onclick="OnClose" class="m-sheet-dialog-header d-flex align-center justify-space-between px-3" style="@($"height:{HeaderHeight}px;")">
+                <div @onclick="OnClose" stopPropagation class="m-sheet-dialog-header d-flex align-center justify-space-between px-3" style="@($"height:{HeaderHeight}px;")">
```
Comment (capdiem) [inline src/Masa.Stack.Components/Shared/IntegrationComponents/SSheetDialog.razor:8]: `@onclick:stopPropgation`

**Review by capdiem — CHANGES_REQUESTED** [review]

**Comment by zyz** [comment]
1.Remove the debug info.
2.stopPropgation to @onclick:stopPropgation
[resolved]

**Review by doddgu — APPROVED** [review]

**Review by capdiem — APPROVED** [review]

**Pull request merged**